src/openllm/models/baichuan/modeling_baichuan.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2023 BentoML Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import annotations
+import typing as t
+
+import openllm
+
+
+if t.TYPE_CHECKING:
+    import torch
+
+    import bentoml
+    import transformers
+else:
+    torch = openllm.utils.LazyLoader("torch", globals(), "torch")
+    transformers = openllm.utils.LazyLoader("transformers", globals(), "transformers")
+
+
+class Baichuan(openllm.LLM["transformers.PreTrainedModel", "transformers.PreTrainedTokenizerBase"]):
+    __openllm_internal__ = True
+
+    def llm_post_init(self):
+        self.device = torch.device("cuda")
+
+    def load_model(self, tag: bentoml.Tag, *args: t.Any, **attrs: t.Any) -> t.Any:
+        trust_remote_code = attrs.pop("trust_remote_code", True)
+        return transformers.AutoModelForCausalLM.from_pretrained(
+            self.model_id, trust_remote_code=trust_remote_code, **attrs
+        )
+
+    def load_tokenizer(self, tag: bentoml.Tag, **attrs: t.Any) -> t.Any:
```

Comment:
U dont need to do this, since you already set trust_remote_code under BaichuanConfig. Unless there are special way to load this tokenizer, load_tokenizer js not needed here.

Reply:
Gotcha!

Reply:
I've removed these two load functions. Thanks!